### PR TITLE
Changed the color of Add-Patient Button

### PR DESCRIPTION
**Pull request by ujjwaltyagi355**
Hello,
I have changed the color of the Add-Patient Icon to white from blue.
Thank You.

### CODE REVIEW

File: src/add-patient-link.tsx [hunk 1]
```diff
@@ -2,12 +2,11 @@ import React from 'react';
 import { ConfigurableLink } from '@openmrs/esm-framework';
 import Add20 from '@carbon/icons-react/es/add/20';
 import { HeaderGlobalAction } from 'carbon-components-react/es/components/UIShell';
-import './add-patient-link.css';
 export default function Root() {
   return (
     <HeaderGlobalAction aria-label="Add" aria-labelledby="Add Patient" name="AddPatientIcon">
-      <ConfigurableLink to="${openmrsSpaBase}/patient-registration">
-        <Add20 className="add-patient-color" />
+      <ConfigurableLink style={{ color: '#FFFFFF' }} to="${openmrsSpaBase}/patient-registration">
```
Comment (HerbertYiga) [inline src/add-patient-link.tsx:8]: looks good,did we have a ticket Id for this fix?

Comment (ujjwaltyagi355) [reply src/add-patient-link.tsx:8]: Hello, @HerbertYiga,
This is related to the [MF-464](https://issues.openmrs.org/browse/MF-464), to add the Add-Patient button on the primary navbar.
I did that but you would have noticed that the color for that was blue.
So this is to make the color white for that.
Thank you.

**Review by FlorianRappl — APPROVED** [review]
LGTM! Thanks

**Comment by ujjwaltyagi355** [comment]
Hello, @FlorianRappl,
The Add-Icon is not getting rendered the same way as it is in my local dev.

The open-mrs site shows like:
image

But my local dev shows:
image

Can you help me with this?
Thank you

**Comment by FlorianRappl** [comment]
> Can you help me with this?

Sure. Please post the computed style that you get locally. Compare them to the online version.

**Comment by ujjwaltyagi355** [comment]
Hello, @FlorianRappl
**tl;dr**
I tried changing the styles through my console for the online version and found out the desired result could be obtained using:
1. Setting the "background-color" in "bx--btn--primary" to none.
2. Changing the margin for the add-button icon to 10px.

But I think there is some other issue apart from this, because the local env shows the desired result but the online site differs. So please have a look at the styling.
Thank you.
### The computed styles for the outer box **online version** are:
image
image
image

### Styling for the outer box in the **local env**:
image
image


### For the add-button on the online version the styling is:
image
image
image
image

### Styling for add-button in the Local Env:
image
image

I really appreciate your patience with this.
 Thank you.

**Comment by ujjwaltyagi355** [comment]
@FlorianRappl,
I would be submitting a new PR by changing the configurable link to navigate to solve this issue.
Thank you for all the help @FlorianRappl.
Really appreciate it.
